Programs/_freeze_module.c

```diff
@@ -121,6 +121,7 @@ compile_and_marshal(const char *name, const char *text)
         return NULL;
     }
 
+    assert(Py_MARSHAL_VERSION >= 5);
     PyObject *marshalled = PyMarshal_WriteObjectToString(code, Py_MARSHAL_VERSION);
```

Comment:
In the Makefile, `_freeze_module` doesn't depend on `marshal.h`, but needs to be rebuilt when `Py_MARSHAL_VERSION` switches to 5.
Instead of properly changing the Makefile, which would be *another* rabbit hole, I decided to touch `_freeze_module.c` instead.